autogpt_platform/backend/backend/blocks/io.py

```diff
@@ -550,3 +550,17 @@ def __init__(self):
                 ("result", False),
             ],
         )
+
+
+IO_BLOCK_IDs = [
+    AgentInputBlock().id,
+    AgentOutputBlock().id,
+    AgentShortTextInputBlock().id,
+    AgentLongTextInputBlock().id,
+    AgentNumberInputBlock().id,
+    AgentDateInputBlock().id,
```

Comment:
You can get all blocks and filter by the block type

Reply:
Downside of that is we have to load all block modules and initialize all 250 blocks instead of just these 10. This is much more efficient.

Reply:
Let me know what you think. I have your proposed change in a stash locally so I can apply it if you want me to.

Reply:
I think loading them all once per startup is fine.

Ironically, all the usage of `get_blocks` initiates the block immediately right after the call, which basically we can just cache the initialized block instead of the class, where we can do later if it actually matters.

Reply:
I've [tried](https://github.com/Significant-Gravitas/AutoGPT/pull/9695/commits/71f0cf9f0000b5c714e48e3a912ac1ec8ad79939) but it gives a lot of circular import issues